proto/encore/engine/trace/trace.proto

```diff
@@ -36,9 +36,11 @@ message Request {
   string endpoint_name = 18;
 
   // Fields set if Type == PUBSUB_MSG
-  string message_id = 19;
-  uint32 attempt = 20;
-  uint64 publish_time = 21;
+  string topic_name = 19;
+  string subscription_name = 20;
```

Comment:
These are needed for the platform's trace view, as we don't have access to the defloc's to extra this in the UI like the local dev does